File: Analisis_data_shopee/pembelian/5.py
import pandas as pd
import mysql.connector
import numpy as np
import matplotlib.pyplot as plt

connection = mysql.connector.connect(host='localhost', user='root', password='', database='pembelian')
sql1 = "SELECT * FROM salesman"
df = pd.read_sql(sql1, connection)
df = df[['salesman_name', 'commision']]
df = df.sort_values('commision', ascending=False).reset_index(drop=True)
print(df.head(1))

File: Analisis_data_shopee/pembelian/5.r
library(RMySQL)
library(dplyr)
library(lubridate)

# Ganti 'dbname', 'username', 'password', dan 'host' sesuai dengan informasi database Anda
con <- dbConnect(MySQL(), dbname = "pembelian", user = "root", password = "", host = "localhost")

query <- "SELECT * FROM salesman"
df <- dbGetQuery(con, query)
dbDisconnect(con)

print(df)

hasil <- select(df, c(salesman_name, commision))
hasil <- select(df, salesman_name, commision)
print(hasil)
hasil <- arrange(hasil, desc(commision))
print(hasil)
print(head(hasil, 1))